storybook/preview.js

```diff
@@ -15,6 +15,82 @@ import cem from './custom-elements.json';
 
 setCustomElementsManifest(cem);
 
+export const globalTypes = {
+    system: {
+        title: 'Design context',
+        description: 'The variation of Spectrum to use in the component',
+        defaultValue: 'spectrum',
+        type: 'string',
+        showName: true,
+        toolbar: {
+            items: [
+                { value: 'spectrum', title: 'Spectrum 2', right: 'default' },
+                { value: 'legacy', title: 'Spectrum 1', right: 'legacy' },
+                { value: 'express', title: 'Express' },
+            ],
+            dynamicTitle: true,
+        },
+    },
+    color: {
+        title: 'Color',
+        description: 'Controls the color context of the component',
+        defaultValue: 'light',
+        icon: 'paintbrush',
+        type: 'string',
+        toolbar: {
+            items: [
+                { value: 'light', title: 'Light', right: 'default' },
+                { value: 'dark', title: 'Dark' },
+            ],
+            dynamicTitle: true,
+        },
+    },
+    scale: {
+        title: 'Platform scale',
+        description: 'Controls the platform scale of the component',
+        defaultValue: 'medium',
+        type: 'string',
```

Comment:
if we miss setting the correct type of these parameters does it fail build time or at run time? Asking since its a bit too much of a boilerplate?

Reply:
You don't have to touch these except when you're changing the global arg types in storybook and it's well documented on their website.